Rebus.AdoNet.Tests/SagaPersisterTests.cs

```diff
@@ -248,7 +329,7 @@ public void InsertDoesPersistSagaData()
 		[Test]
 		public void WhenIgnoringNullProperties_DoesNotSaveNullPropertiesOnUpdate()
 		{
-			var persister = CreatePersister(createTables: true);
+			var persister = _persister;
```

Comment:
let's get rid of this.. use _persister directly.

Reply:
ack